dipy/sims/voxel.py

```diff
@@ -288,7 +299,265 @@ def multi_tensor(gtab, mevals, S0=100, angles=[(0, 0), (90, 0)],
                                                  snr=None)
 
     return add_noise(S, snr, S0), sticks
+    
+def multi_tensor_dki(gtab, mevals, S0=100, angles=[(0., 0.), (90., 0.)],
+                         fractions=[50, 50], snr=20):
+    
+    r"""Simulate the elements of the diffusion and diffusion kurtosis tensor as
+    well as the diffusion signal S based on the DKI model (i.e. taylor 
+    accumulante larger than the fourth order are ignored). Simulations are 
+    based on multicompartmental models which assumes that tissue are described
+    by impermeable diffusion compartments characterized by their only diffusion
+    tensor.
+
+    Parameters
+    -----------
+    gtab : GradientTable
+    mevals : array (K, 3)
+        eigenvalues of the diffusion tensor for each individual compartment
+    S0 : float
```

Comment:
Mark optional arguments as "(optional)" and tell us what the default values are.